code-studio/src/js/pd/workshop_dashboard/workshop_dashboard.jsx

```diff
@@ -0,0 +1,55 @@
+/* global React WORKSHOP_CONSTANTS */
+
+const Router = require('react-router').Router;
+const Route = require('react-router').Route;
+const IndexRedirect = require('react-router').IndexRedirect;
+const useRouterHistory = require('react-router').useRouterHistory;
+const createHistory = require('history').createHistory;
+const NewWorkshop = require('./new_workshop.jsx');
+const Workshop = require('./workshop.jsx');
+const Header = require('./header.jsx');
+const WorkshopIndex = require('./workshop_index.jsx');
+const WorkshopAttendance = require('./attendance/workshop_attendance.jsx');
```

Comment:
nit/fyi - if you're using ES6 you can use import instead of requiring as consts, i.e.
```
import { Router, Route, IndexRedirect, useRouterHistory, createHistory } from 'react-router';
import NewWorkshop from './new_workshop';
...
```